README.md

```diff
@@ -343,6 +346,14 @@ ONLY:
     - prod
 server-tools:
   - auth_*
+---
+# Custom repositories
+ENV:
+  DEFAULT_REPO_PATTERN: https://github.com/it-projects-llc/{}.git
```

Comment:
I would prefer here to not point to other specific repo and let documentation "brand agnostic".

Reply:
I changed it to Tecnativa, as it's already used in the README
https://github.com/Tecnativa/doodba/compare/1d563c5050a228bb97fa52459dfa7c8f7280760e..1a5a25f3e1123c4c2e52793ba45e79097fead486

Reply:
Well, it's not needed to put `Tecnativa`. I was thinking in a generic `team` word, but thanks for branding us.

Reply:
I decided that github may not have something like ``example.com`` and any generic ``team`` may be occupied later

Reply:
OK, thanks